requirements.txt

```diff
@@ -6,4 +6,4 @@ djangorestframework>=3.8.2,<3.10.0
 passlib>=1.7.1,<1.8.0
 django-filter>=2.1.0,<2.2.0
 djangorestframework-link-header-pagination>=0.1.1,<0.2.0
-xhtml2pdf>=0.2.3,<0.3.0
+weasyprint>=43,<=50 
```

Comment:
here should be `weasyprint>=43,<51`

Reply:
Ok, but why we should use `<51`? Is it because of subversions of `50` may be released (like `50.1`)?

Reply:
yes, that's what may happen and is usually the case for most libraries